Problem3.py

```diff
@@ -0,0 +1,29 @@
+"""Search in Infinite sorted array"""
+
```

Comment:
Please add time and space complexities along with other required information. You can contact me, the cohort TA or the professor if you had any questions.